index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 module.exports = function (arr) {
-	var rest = [].concat.apply([], [].slice.call(arguments, 1));
-	return arr.filter(function (el) {
-		return rest.indexOf(el) === -1;
+	const rest = new Set(...Array.from(arguments).slice(1));
```

Comment:
Spread is supported since 5 I believe so it should be reverted to what it was. Don't think there is another solution.